gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -351,6 +351,11 @@ if CLIENT then
     -- stylua: ignore
     local cvCrosshairMode = CreateConVar("ttt_crosshair_mode", "0", FCVAR_ARCHIVE)
 
+    ---
+    -- @realm client
+    -- stylua: ignore
+    local cvEnableHUDBoxBlur = GetConVar("ttt2_hud_enable_box_blur")
```

Comment:
```suggestion
    local cvEnableHUDBoxBlur = GetConVar("ttt2_hud_enable_box_blur")
```
These doc strings are not needed. Our Doc-tool only looks for `CreateConVar` calls